### ISSUE TITLE
[epic] Make elemental-operator available in the Rancher marketplace

### ISSUE THREAD

**Issue by kkaempf** [rancher/elemental-operator#376]
Goal:
- enable operator installation via Rancher UI
- enable operator update via Rancher UI
- enable Rancher admin be notified when operator update is available.

Extend the `elemental-operator` helm chart to be accepted in the Rancher marketplace.

Reach out to Epinio/Kubewarden for examples.

- [x] rancher/elemental-operator#492
- [x] rancher/elemental-operator#493
- [x] rancher/elemental-docs#234
- [x] #551
- [x] #552

**Comment by davidcassany** [comment]
* We need the elemental-operator chart published in https://github.com/rancher/charts (note this moves the build out of OBS)
* To introduce a new Rancher package there are some docs [here](https://github.com/rancher/charts/blob/dev-v2.7/docs/developing.md)
* As an example Kubewarden already created an [RFC](https://github.com/kubewarden/rfc/blob/main/rfc/0009-rancher-integration-charts.md) about how to include it as a Rancher package.
* Rancher also includes the concept of "Questions", which is used to enable installation UI including custom options. Obviously this requires some specific template to make use of it.
* Some additional docs can be found in confluence.

Basically all this results into creating a chart following some strict templates and practices. We need to check how far are we from the requested format.

**Comment by fgiudici** [comment]
Steps proposal:
- open a PR for the current stable version 1.2.2 (draft branch [here](https://github.com/fgiudici/rancher-charts/tree/add_elemental))
- add questions.yml ([reference](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/helm-charts-in-rancher/create-apps#additional-files-for-rancher-charts))
- update to a new release when we will have the IBS links

**Comment by fgiudici** [comment]
https://github.com/rancher/charts/pull/2658

**Comment by kkaempf** [comment]
Waiting for the next Elemental release (with properly splitted charts)

**Comment by fgiudici** [comment]
PR on Rancher Charts updated (https://github.com/rancher/charts/pull/2658).
Now we have the splitted charts there, should allow upgrades or install from scratch.
Reviewed annotations, updated README and bell and whistles (questions.yaml making nice UI for picking up OS channel selection and enabling debug if needed).

The chart can be tested before the marge in the marketplace by adding the temporary repository of the pull request as a *github repo* in Rancher UI:
```
Git Repo URL: https://github.com/fgiudici/rancher-charts
Git Branch: add_elemental
```
I tested it myself, all looks good, but would be good to get some more 👀   and testing 😄 

The charts could also be downloaded for inspection or installed via the cli using the URLs:
https://github.com/fgiudici/rancher-charts/raw/add_elemental/assets/elemental-crd/elemental-crd-102.0.0%2Bup1.2.5.tgz
https://github.com/fgiudici/rancher-charts/raw/add_elemental/assets/elemental/elemental-102.0.0%2Bup1.2.5.tgz
